packages/maskbook/src/social-network-provider/twitter.com/utils/selector.ts

```diff
@@ -26,10 +26,10 @@ export const postEditorContentInPopupSelector: () => LiveSelector<E, true> = ()
     querySelector<E>('[aria-labelledby="modal-header"] > div:first-child > div:nth-child(3)')
 export const postEditorInPopupSelector: () => LiveSelector<E, true> = () =>
     querySelector<E>(
-        '[aria-labelledby="modal-header"] > div:first-child > div:nth-child(3) > div:first-child > div:first-child',
+        '[aria-labelledby="modal-header"] > div:first-child > div:nth-child(3) > div:first-child > div:first-child [role="button"][aria-label]:nth-child(6)',
     )
 export const postEditorInTimelineSelector: () => LiveSelector<E, true> = () =>
-    querySelector<E>('[role="main"] :not(aside) > [role="progressbar"] ~ div')
+    querySelector<E>('[role="main"] :not(aside) > [role="progressbar"] ~ div [role="button"][aria-label]:nth-child(6)')
```

Comment:
`:last-child` and `:last-of-type` do not work well, so use `:nth-child(6)`

Reply:
not work cause it will include a dead loop?

Reply:
`:last-of-type` selects a wrong element. `:last-child` includes a dead loop.